lib/Jsrt/Core/JsrtCore.cpp

```diff
@@ -1392,6 +1392,22 @@ CHAKRA_API JsAllocRawData(_In_ JsRuntimeHandle runtimeHandle, _In_ size_t sizeIn
     });
 }
 
+CHAKRA_API JsIsCallable(_In_ JsValueRef object, _Out_ bool *isCallable)
+{
+    return ContextAPIWrapper<JSRT_MAYBE_TRUE>([&](Js::ScriptContext *scriptContext, TTDRecorder& _actionEntryPopper) -> JsErrorCode {
```

Comment:
>ContextAPIWrapper [](start = 11, length = 17)

do we need this wrapper?

Reply:
Good call. I also removed it from JsIsConstructor

Reply:
nice

---
In reply to: [264460121](https://github.com/Microsoft/ChakraCore/pull/6012#discussion_r264460121) [](ancestors = 264460121)